recipes/sphinx/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set version = "1.4.5" %}
+{% set sha256 = "c5df65d97a58365cbf4ea10212186a9a45d89c61ed2c071de6090cdf9ddb4028" %}
+
+package:
+  name: sphinx
+  version: {{ version }}
+
+source:
+  fn: Sphinx-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/S/Sphinx/Sphinx-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - six
+    - jinja2
+    - pygments
+    - docutils
+    - snowballstemmer
+    - babel
+    - alabaster
+    - imagesize
+    - requests
+    - colorama  # [win]
+
+test:
+  commands:
+    - sphinx-quickstart --version
+
+about:
+  home: http://www.sphinx-doc.org
+  license: BSD
```

Comment:
Seems to be `BSD 2-clause`, see [here](https://github.com/sphinx-doc/sphinx/blob/master/LICENSE). Should this be added?